mixnet/node/src/config.rs

```diff
@@ -26,11 +36,25 @@ impl Default for MixnetNodeConfig {
             )),
             private_key: PrivateKey::new().to_bytes(),
             connection_pool_size: 255,
+            max_retries: 3,
+            retry_delay: Duration::from_secs(5),
         }
     }
 }
 
 impl MixnetNodeConfig {
+    const fn default_connection_pool_size() -> usize {
```

Comment:
could they be consts?

Reply:
`serde(default = "...")` does not allow to use consts, have to use a fn.